### PR TITLE
Bug 1455381 - Add normalized appName and os fields to mobile pings [WiP]

### PR DESCRIPTION
**Pull request by akkomar**
As per [Bug 1455381](https://bugzilla.mozilla.org/show_bug.cgi?id=1455381) we'd like to have normalized os and application names from mobile pings for monitoring via CEP plugin and partitioning.

### CODE REVIEW

**Comment by akkomar** [comment]
@trink could you please briefly take a look if the approach here is correct?
Application and os list still needs to be finalized, for now I'd like to make sure I'm implementing this in the right place.

**Comment by akkomar** [comment]
@fbertsch yes, I was planning to first have this working and tested with hardcoded values (as I haven't touched this project before) and then make it configurable.

**Comment by fbertsch** [comment]
Great! Very exciting. Thanks for the info @akkomar

File: moz_telemetry/io_modules/decoders/moz_ingest/telemetry.lua [hunk 1]
```diff
@@ -252,14 +252,18 @@ local function process_json(hsr, msg)
...
         local sourceVersion = doc:value(doc:find("v"))
         validate_schema(hsr, msg, doc, sourceVersion)
         msg.Fields.sourceVersion = tostring(sourceVersion)
         clientId = doc:value(doc:find("clientId"))
         msg.Fields.clientId = clientId
         submissionField.value = doc
         msg.Fields.submission = submissionField
+
+        -- Normalize os and appName
+        msg.Fields.normalizedOs = mtn.mobile_os(doc:value(doc:find("os")))
```
[the remark this answers is not in the document]
Comment (akkomar) [reply moz_telemetry/io_modules/decoders/moz_ingest/telemetry.lua]: Hey @trink, so I'm writing integration test [1]. IIUC I should be verifying if the mobile ping message is correctly enriched in [2].

However, I'm not sure how do I add a new ping to test data. It looks like tests are ingesting serialized pings from [3], and this dump seems to be produced with [4] - is that correct?
How do I repeat the process of updating [3]?

[1] https://github.com/akkomar/lua_sandbox_extensions/commit/f70509b22a355c3dbaf6488c30a4457714a6ff89
[2] https://github.com/mozilla-services/lua_sandbox_extensions/blob/0e839716db2715f662a05e210c9d11ab5c810b02/moz_telemetry/tests/integration/decoder_telemetry/run/analysis/verify_decoder.lua#L8
[3] https://github.com/mozilla-services/lua_sandbox_extensions/blob/master/moz_telemetry/tests/integration/decoder_telemetry/input.hpb
[4] https://github.com/mozilla-services/lua_sandbox_extensions/blob/master/moz_telemetry/tests/integration/decoder_telemetry/run/input/generate_data.lua

[the remark this answers is not in the document]
Comment (trink) [reply moz_telemetry/io_modules/decoders/moz_ingest/telemetry.lua]: It is a manual process, run the generate_data plugin locally and overwrite input.hpb with the new output (output/input/0.log), test and then commit.

**Pull request merged**